usaspending_api/etl/transaction_loaders/tests/test_fpds_loader.py

```diff
@@ -209,60 +201,33 @@ def test_load_from_broker():
 # This only runs for one of the most simple tables, since this is supposed to be a test to ensure that the loader works,
 # NOT that the map is accurate
 def test_create_load_object(monkeypatch):
-    custom_bools = {"is_awesome": "awesome", "is_not_awesome": "nawesome", "reasons_to_not_be_awesome": "noawersn"}
+    columns = {"input_field_1": "output_field_1", "input_field_2": "output_field_2"}
+    bools = {"input_field_3": "output_field_3", "input_field_4": "output_field_4", "input_field_5": "output_field_5"}
+    functions = {"input_field_6": lambda t: t["input_field_6"] * 2, "input_field_7": lambda t: "replacement"}
```

Comment:
```suggestion
    functions = {"output_field_6": lambda t: t["input_field_6"] * 2, "output_field_7": lambda t: "replacement"}
```
Functions are backwards relative to the other maps, probably a mistake.